main.go

```diff
@@ -114,15 +114,22 @@ func main2() error {
 				// Ignore (https://github.com/go-httprequest/httprequest/blob/2b21a94c9e788981d4e609ef4b7a21cedae6da66/type.go#L225)
 				continue
 			}
+			name := strcase.ToCamel(op.OperationID + "Request")
 			req := templates.Definition{
-				Name: strcase.ToCamel(op.OperationID + "Request"),
+				Name:       name,
+				DocComment: templates.Comment(fmt.Sprintf("%s holds the request to a %s call.", name, op.OperationID)),
```

Comment:
%s holds the request value for a %s call.

?

Reply:
What do you mean by this?

Reply:
I'm suggesting an alternative wording for the comment.

"FooRequest holds the request to a Foo call" doesn't quite read right to me.